cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -753,636 +835,82 @@ IOResult<void> set_vaccination_data(std::vector<Model<FP>>& model, const std::st
     * @param divi_data_path path to divi data file
     * @param confirmed_cases_path path to confirmed cases file
     * @param population_data_path path to population data file
+    * @param set_vaccination_data boolean to set vaccination data
+    * @param vaccination_data_path path to vaccination data file
```

Comment:
Please also adapt all the other parameters in the documentation eg models instead of model and dir instead of data_dir, counties instead of region...

You should again also describe which files are produced.